### ISSUE TITLE
angular-split has a new maintainer! (was: Sunsetting angular-split)

### ISSUE THREAD

**Issue by beeman** [angular-split/angular-split#369]
Hi everyone,

Just a heads-up: I'm sunsetting the project angular-split. I haven't been using this library in my own projects for some time, and maintaining it is no longer feasible for me.

The code is open source and available for anyone to fork. If you're experienced in maintaining open-source libraries (with visible activity on GitHub/NPM), feel free to reach out on Discord or Twitter (links in my profile) about taking over. Don't bother applying if you are not actively maintaining any projects as I don't have time to guide you.

Thanks for your support and contributions over the years!

**Comment by beeman** [comment]
Hey there!

I'm excited to announce @Jefiozie has reached out to me he and @SanderElias are willing to take over maintenance of this project! 🥳

I think this is great news for everybody using the plugin and I hope it'll become a better maintained library soon!

**Comment by Jefiozie** [comment]
Hello Angular-Split Community 👋,

Exciting news! We, @SanderElias and @Jefiozie , are the new maintainers of Angular-Split. First off, a huge thank you to @beeman  for his exceptional work and dedication in leading this project.

**Introducing Ourselves:**

- @SanderElias : With over 40 years of experience in the technology sector, I began my journey with Angular during its early beta stages. As a Google Developer Expert (GDE) for Angular and web technologies, I have initiated and contributed to various open-source projects. I am the creator of Scully and actively engage in community events, sharing my expertise and insights with others in the field.
- @Jefiozie : Over the past few years, I've dedicated myself to working with Angular, during which I've initiated several open source projects. My most enduring project is ngx-aws-deploy, enabling the deployment of Angular apps to Amazon S3 storage. Additionally, in collaboration with Sander, I recently developed Validointi, a library designed to enhance template forms using Vestjs. My contributions also extend to NGRX, the Angular repository, Scully, and various other projects.

We're committed to transparency and look forward to sharing our plans. Stay tuned for a GitHub Discussion link where we'll outline our upcoming initiatives and welcome your input.

Looking forward to a bright future with the Angular-Split community!
